### DISCUSSION TITLE
OpenTofu webUI SelfService

### DISCUSSION THREAD

**Discussion by D10S0VSkY-OSS** [opentofu/opentofu#961] [General]
Selft Service UI & rest api for  OpenTofu automation ( Like terraform cloud ) [OpenTofu UI](https://github.com/D10S0VSkY-OSS/Stack-Lifecycle-Deployment)
![image](https://github.com/opentofu/opentofu/assets/79284025/f230ca3c-d0a0-444a-a6cc-cefb77f9b131)
![image](https://github.com/opentofu/opentofu/assets/79284025/1e70d9cb-e8cb-4e18-b350-6c0fc41c1977)

**Comment by eavichay** [comment]
This is awesome work, where can we talk? Is there a slack channel I can reach out to you?
